src/tck/src/SdkClient.cc

```diff
@@ -129,6 +136,176 @@ nlohmann::json SdkClient::createAccount(const std::optional<std::string>& key,
   };
 }
 
+//-----
+nlohmann::json SdkClient::createToken(const std::optional<std::string>& name,
+                                      const std::optional<std::string>& symbol,
+                                      const std::optional<uint32_t>& decimals,
+                                      const std::optional<uint64_t>& initialSupply,
+                                      const std::optional<std::string>& treasuryAccountId,
+                                      const std::optional<std::string>& adminKey,
+                                      const std::optional<std::string>& kycKey,
+                                      const std::optional<std::string>& freezeKey,
+                                      const std::optional<std::string>& wipeKey,
+                                      const std::optional<std::string>& supplyKey,
+                                      const std::optional<bool>& freezeDefault,
+                                      const std::optional<int64_t>& expirationTime,
+                                      const std::optional<std::string>& autoRenewAccountId,
+                                      const std::optional<int64_t>& autoRenewPeriod,
+                                      const std::optional<std::string>& memo,
+                                      const std::optional<std::string>& tokenType,
+                                      const std::optional<std::string>& supplyType,
+                                      const std::optional<int64_t>& maxSupply,
+                                      const std::optional<std::string>& feeScheduleKey,
+                                      const std::optional<std::vector<std::shared_ptr<CustomFee>>>& customFees,
+                                      const std::optional<std::string>& pauseKey,
+                                      const std::optional<std::string>& metadata,
+                                      const std::optional<std::string>& metadataKey,
```

Comment:
Was thinking about the parameters here and for the other `SdkClient` functions.. Won't it look better to have them defined in something like `CommonTransactionParams` but for example `CreateTokenTransactionParams`. This way we won't have to write those in both the `.h` and `.cc` or potentially somewhere else as arguments. Also this way we would only pass in one reference to the `CreateTokenTransactionParams`. WDYT?

Reply:
Yesss I actually had this same thought recently as well, I think it would help clean up files and make it less error-prone. I was planning on fixing this for all of our functions, as well as dividing up the `SdkClient` into more manageable files as well.